### PR TITLE
Update deps catch2 version to 2.13.9

### PR DESCRIPTION
**Pull request by h3poteto**
I update catch version to 2.13.9.
I'd like to import [this change](https://github.com/catchorg/Catch2/issues/2178) because I can't build this repository.

### CODE REVIEW

File: scripts/get-dep.sh [hunk 1]
```diff
@@ -57,8 +57,8 @@ function get_json()
 
 function get_catch()
 {
-	GIT_REPO="https://github.com/philsquared/Catch.git"
-	GIT_TAG="v2.11.0"
+	GIT_REPO="https://github.com/catchorg/Catch2.git"
```
Comment (h3poteto) [inline scripts/get-dep.sh:60]: `https://github.com/philsquared/Catch` is renamed. At the moment, the catch repository URL is `https://github.com/catchorg/Catch2`.

**Comment by ibc** [comment]
Thanks, will merge and release tomorrow.

**Comment by h3poteto** [comment]
When do you merge this?
Is there a problem?

**Review by ibc — APPROVED** [review]

**Comment by h3poteto** [comment]
@ibc Could you please release a new version?

**Comment by ibc** [comment]
libsdptransform 1.2.10 released.

**Pull request merged**